### PR TITLE
Arrayhandling

### LINKED ISSUE #16: Additional wildcards handling
Facilitate for returning arrays when wildcards are submitted (chp. 3.6 of OpenServer manual): 
- Underscore (_)
- @ Sign and *
- @ Sign and ?
- @ Sign and #
- @ Sign and !

### PR DESCRIPTION
**Pull request by smolvik1**
Fixes #16

### CODE REVIEW

**Comment by smolvik1** [comment]
All 15 pytest testing passed

File: openserver/openserver.py [hunk 1]
```diff
@@ -99,8 +99,13 @@ def DoGet(self, Gv):
             if value.isdigit():  # Checking if integer
                 value = int(value)
             elif '|' in value:  # Checking if | in string is returned
-                if any(x in Gv for x in (',', '[$]')):
-                    value = np.fromstring(value, sep="|")
+                if any(x in Gv for x in (',', '[$]', '@', ':', '_')):
```
Comment (knudsvik) [inline openserver/openserver.py]: We are only testing DoGet with `[$]` and `[1:3]`. If `,`, `@` and `_` are valid options, they should be tested too, agreed?

**Review by knudsvik — CHANGES_REQUESTED** [review]

File: openserver/openserver.py (hunk 1 above)
Comment (smolvik1) [reply openserver/openserver.py]: Removed '_' from wildcard check as it only retrieves the last item, never an array. 
Added two (2) testing functions, one for ',' and one for '@'. The latter relied on GAP model for testing, so gap_testfile added for this purpose.

**Comment by smolvik1** [comment]
All 17 pytest tests passed

**Review by knudsvik — APPROVED** [review]

**Pull request merged**